### PR TITLE
Update documentation and travis

### PR DESCRIPTION
**Pull request by aliak00**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,23 +5,13 @@
...
-
-    1. **Option 2:** If you want to just be able to build and test with Xcode, install [CocoaPods](https://cocoapods.org/) by running in the terminal:
-
-            sudo gem install cocoapods
-            pod setup --verbose
-
-        If there's a problem in [Podfile.lock](Example/Podfile.lock) having unquoted paths, update psych:
-
-            sudo gem install psych -v 2.2.2
+    **Note**: Installing a ruby version manager such as [RVM](https://rvm.io) will make life a little easier
```
Comment (lrossi) [inline CONTRIBUTING.md]: But it might break App Store submission 😜

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -93,14 +76,14 @@ For example:
...
 * `Removed: Removed deprecated Manager.init(...) overloads.`
 * `Fixed: Fixes a bug where client tokens would not be refreshed correctly.`
-* `Security: Fixes a concern where user credentials were vulnerable to attacks based on redirects.
+* `Security: Fixes a concern where user credentials were vulnerable to attacks based on redirects.`
 
 ## **Running the tests**
 
-In XCode: run the tests using &#8984;U (Product - Test).
-Make sure that the "SchibstedAccount-Example" scheme is selected.
+In XCode: run the tests using &#8984;U (Product - Test). Make sure that the "Example" scheme is selected.
```
Comment (lrossi) [inline CONTRIBUTING.md]: `Xcode` 😇

File: scripts/format.sh [hunk 3]
```diff
@@ -1,15 +1,31 @@
...
-        --header "\
+set -e
+
+copyright="\
 \n\
  Copyright 2011 - 2018 Schibsted Products & Technology AS.\n\
  Licensed under the terms of the MIT license. See LICENSE in the project root.\n\
 "
+common_args=("Source/" "Example/"
+        "--exclude" "Example/Pods/"
```
Comment (lrossi) [inline scripts/format.sh]: Is this still relevant?

File: CONTRIBUTING.md (hunk 1 above)
Comment (aliak00) [reply CONTRIBUTING.md]: Haha 😭

File: scripts/format.sh (hunk 3 above)
Comment (aliak00) [reply scripts/format.sh]: Nah, will remove

File: CONTRIBUTING.md (hunk 2 above)
Comment (aliak00) [reply CONTRIBUTING.md]: 😆 👍

**Pull request merged**